Documentation/Books/Manual/Security/ChangeRootPassword.md

```diff
@@ -0,0 +1,32 @@
+# Howto Reset Root Password
+
+One can reset Root Password in the following way:
+
+1) Stop the server
+2) Set `authentication=false` in the `arangod.conf` file
+3) Restart the server
```

Comment:
We should recommend to use localhost for the binding (`--server.endpoint tcp://127.0.0.1:8529`) at least for single server, because you don't want to accidentally expose your server without authentication to any network (other than a LAN perhaps). Not sure about other deployment modes.